code/game/machinery/status_display.dm

```diff
@@ -127,7 +127,9 @@
 	if(overlay)
 		qdel(overlay)
 
-	return new/obj/effect/overlay/status_display_text(src, line_y, message)
+	var/obj/effect/overlay/status_display_text/new_status_display_text = new/obj/effect/overlay/status_display_text(line_y, message)
```

Comment:
this syntax gives me hives

Reply:
good point i didn't even notice i did that